Entities/ACraft.cpp

```diff
@@ -792,6 +792,17 @@ void ACraft::DropAllInventory()
     }
 }
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+float ACraft::GetNewInventoryMass() const {
```

Comment:
I think it'd be good to change NewInventory to AddedInventory to match up with MovableMan.AddedActors/AddedItems/etc.

Reply:
I think NewInventory makes more sense in this case! The "Added" prefix reminds me of objects that were added on this very frame, which doesn't exactly match the NewInventory's purpose in my head.

Reply:
Oh, right, cause NewInventory isn't moved over until the doors close. I guess my real problem with it is that in my head I read it as Get**New**InventoryMass, as if it's getting the newInventoryMass as opposed to the oldInventoryMass. Obviously that's not a thing and it doesn't make sense, but I think that's where my concerns stemmed from.

Maybe there's no better wording though, I dunno. @MaximDude any thoughts?

Reply:
Maybe `CollectedInventoryMass` or `AbductedCargoMass`? idk.
`NewInventory` definitely doesn't sound right.

Reply:
`CollectedInventory` or `CollectInventory` sounds good to me, @garethyr?

Reply:
I'm cool with that